.azure-pipelines/bazel.yml

```diff
@@ -51,6 +51,10 @@ steps:
       SLACK_TOKEN: $(SLACK_TOKEN)
       REPO_URI: $(Build.Repository.Uri)
       BUILD_URI: $(Build.BuildUri)
+      BUILD_REASON: $(Build.Reason)
```

Comment:
To add this for macos please add here! https://github.com/envoyproxy/envoy/blob/da1b0e4b9372a28519c60c336da0a5fbb20b617b/.azure-pipelines/pipelines.yml#L371

And windows here: https://github.com/envoyproxy/envoy/blob/da1b0e4b9372a28519c60c336da0a5fbb20b617b/.azure-pipelines/pipelines.yml#L398

Reply:
(sidebar it is unfortunate we have these in all different places, maybe something to refactor later but not part of this PR)

Reply:
This is implicitly implied by AZP itself, (they translate variable names to env vars by convert '.' to '_' and capitalize, so this is already passed into docker in other places already.

Reply:
Awesome, thanks for catching that!